fast64_internal/f3d/f3d_material.py

```diff
@@ -3231,6 +3209,29 @@ class RDPSettings(PropertyGroup):
         update=update_node_values_with_preset,
     )
 
+    @property
+    def using_fog(self):
+        return self.g_fog or any(["FOG" in input for input in self.blend_inputs])
```

Comment:
I personally would just check for the two blend modes directly for the sake of clarity, color inputs will never get new inputs added there is just those two for fog so its good to make that explicit I think
```suggestion
        return self.g_fog or self.does_blender_use_input("G_BL_CLR_FOG") or self.does_blender_use_input("G_BL_A_FOG")
```